package.json

```diff
@@ -7,7 +7,7 @@
   },
   "license": "LicenseRef-LICENSE",
   "scripts": {
-    "postinstall": "ngcc --properties browser main --tsconfig './tsconfig.spec.json'",
+    "postinstall": "ngcc --tsconfig './tsconfig.app.json' --use-program-dependencies && ngcc --source 'node_modules/@angular'",
```

Comment:
What does these new options do?

Reply:
so previously, we were running ngcc across all test dependencies. The issue there is that spectator and spectator jest circularly reference each other (not sure why this wasn't an issue in the past) and break. We don't actually need spectator to be compiled for ivy though - so now we're using `--tsconfig './tsconfig.app.json' --use-program-dependencies` to only compile modules reference from the actual application, and then manually adding `ngcc --source 'node_modules/@angular'` to compile the angular testing modules used by the test bed (and we know all the first party angular ones are compatible with the ivy compiler anyway).